.replit

```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "index.js"
```

Comment:
This may be a dumb question, but shouldn't this be the following for this set of challenges?
```
entrypoint = "server.js"
```